handbook/engineering/deployments/postgresql.md

```diff
@@ -1,25 +1,70 @@
-# PostgreSQL database tips
+# PostgreSQL deployments
 
-### Shell commands
+For deployments other than Cloud and Sourcegraph.com please use the information [here](https://docs.sourcegraph.com/admin/faq#how-do-i-access-the-sourcegraph-database) to access the database.
 
-These commands assume you're on a local machine, and trying to access the live systems. Also refer to the [deployment page's Kubernetes section](./kubernetes.md) for kubectl tips.
+## Sourcegraph.com specific
 
-### Sourcegraph.com specific
+We currently run two seperate databases. The `sg-cloud` database is the primary database and the `sg-cloud-code-intel` is used by the code-intel team.
 
 You can also directly view database in [GCP](https://console.cloud.google.com/sql/instances?project=sourcegraph-dev)
 
-We currently run two seperate databases. The `sg-cloud` database is the primary database and the `sg-cloud-code-intel` is used by the code-intel team.
+We utilize the GCP utility of the [Cloud SQL Proxy](https://cloud.google.com/sql/docs/postgres/sql-proxy) to connect to our production databases. By default, our Cloud SQL databases are not accessible without this proxy.
+
+For read-only access there is also an option of using [BigQuery](
+https://console.cloud.google.com/bigquery?sq=527047051561:67f2616f4acb4b7cb3639e4a97e2f4aa
+) and their `EXTERNAL_QUERY` syntax.
+
+There are two ways of connecting: either using the `gcloud beta sql connect` command or running the proxy on a port locally.
+
+### Command line only use (pgsql)
+
+You may use these gcloud commands to connect directly to the databases:
+
+- Default db {[Password](https://my.1password.com/vaults/dnrhbauihkhjs5ag6vszsme45a/allitems/svfiw4vcbxhhbobpl442olyebu/)}
+  ```
+    gcloud beta sql connect --project sourcegraph-dev sg-cloud-732a936743 --user=sg -d=sg
```

Comment:
Doesn't work for me

```
% gcloud beta sql connect --project sourcegraph-dev sg-cloud-732a936743 --user=sg -d=sg
ERROR: (gcloud.beta.sql.connect) Cloud SQL Proxy could not be found in PATH. See https://cloud.google.com/sql/docs/mysql/sql-proxy#install for information on installing.
```

Reply:
Ahh, I thought it prompted to install the beta utils if you did not have them 
```
You do not currently have this command group installed.  Using it
requires the installation of components: [beta]


Your current Cloud SDK version is: 310.0.0
Installing components from version: 310.0.0

┌─────────────────────────────────────────────┐
│     These components will be installed.     │
├──────────────────────┬────────────┬─────────┤
│         Name         │  Version   │   Size  │
├──────────────────────┼────────────┼─────────┤
│ gcloud Beta Commands │ 2019.05.17 │ < 1 MiB │
└──────────────────────┴────────────┴─────────┘

For the latest full release notes, please visit:
  https://cloud.google.com/sdk/release_notes

Do you want to continue (Y/n)?
```

Reply:
What makes you think I don't have beta tools installed?

Reply:
It seems like the cloud-sql-proxy is not installed as part of the beta tools. I thought it was.